lib/ansible/utils/vars.py

```diff
@@ -230,33 +230,42 @@ def load_options_vars(version):
     return options_vars
 
 
-def isidentifier(ident):
-    """
-    Determines, if string is valid Python identifier using the ast module.
+def isidentifier_PY3(ident):
```

Comment:
Mark these functions as private (leading underscore) since they aren't intended to be invoked directly.

Reply:
Done.